### DISCUSSION TITLE
Refresh button on fields list when editing entry types (and other elements)

### DISCUSSION THREAD

**Discussion by dgtlcrftrs** [craftcms/cms#9039] [Ideas]
### Description

Often I am creating fields whilst editing an entry type and need to do a full refresh of the entry type to see the new fields appear in the fields list (or save if I've also made changes). It would be awesome if there were a little refresh icon next to the fields list to solely refresh it for the speed of adding new fields to entries (or other elements).

**Comment by brandonkelly** [comment]
Related: #2685
